test/unit/api_spec.js

```diff
@@ -2147,6 +2147,25 @@ sozialökonomische Gerechtigkeit.`)
       await loadingTask.destroy();
     });
 
+    it("gets text content, with invisible text mark (issue 9186)", async function () {
```

Comment:
Given that the test-case is linked, you'll need to add the following block to fix the GitHub Actions failure:
```javascript
if (isNodeJS) {
  pending("Linked test-cases are not supported in Node.js.");
}
```